README.md

```diff
@@ -46,7 +46,7 @@ reads the ``setup.py`` file for each package before installing anything).
 
 First install the basic dependencies:
 
-``$ pip install numpy scipy h5py scikit-learn mpi4py``
+``$ pip install numpy scipy h5py scikit-learn``
```

Comment:
I actually think these are no longer required, since all the downstream installs have been fixed. Perhaps `numpy` is still required to be installed first (as its imported in some `setup.py`s), so for now, reduce it to just that.